### ISSUE TITLE
Voices of a 4-Voice-Tune get mixed up when using responsive wrapping

### ISSUE THREAD

**Issue by flreuss** [paulrosen/abcjs#645]
Hi,
I encountered an issue on trying to wrap a tune with 4 separate voices responsively.
On a mobile device, in the second line of the tune the voices get mixed up.
I verified this by calling makeVoicesArray() on the unwrapped version as well as on the wrapped version.

The tune should look like this:
![grafik](https://user-images.githubusercontent.com/79400384/111963243-92d43980-8af3-11eb-9e03-4e8563fa5b83.png)

But on mobile devices (in the picture: 320x568) it ends up looking like this:
![grafik](https://user-images.githubusercontent.com/79400384/111963634-07a77380-8af4-11eb-9916-7d897ab47110.png)

I am using abcjs inside of a NextJS project with the following RenderAbc options:
```
{
  "add_classes": true,
  "paddingbottom": 0,
  "paddingright": 50,
  "paddingleft":15,
  "paddingtop": 15,
  "responsive": "resize",
  "selectionColor": "#7D4CDB",
  "header_only": false,
  "stop_on_warning": false,
  "wrap": { "minSpacing": 1.8, "maxSpacing": 2.7, "preferredMeasuresPerLine": 4 }
}
```

I set `config.staffwidth = window.innerWidth.width` on small devices and `config.staffwidth = window.innerWidth.width / 2` on large devices for better display.

This is the tune in abcnotation:
```
X:0
T:Des Goldschmieds Töchterlein
C:Volkslied, 16. Jh. (Satz nach Robert Kahn)
%%score { ( 1 2 ) | ( 3 4 ) }
L:1/4
M:4/4
I:linebreak $
K:Bb
Q:1/4=120
V:1 treble
V:2 treble 
V:3 bass 
V:4 bass 
V:1
 G A B G | ^F A G B | A G F z |] %3
V:2
 D F F E | D ^F G =F | F B,/C/ D z |] %3
V:3
 B, C D C/B,/ | A, D B, D | D G,/A,/ B, z |] %3
V:4
 G, F, B,, "_D-7"C, | D, D, "_T"G, B,,/C,/ | D, E, B,, z |] %3
```

**Comment by paulrosen** [comment]
Thanks for the bug report. I'll look into this soon.

**Comment by watersoft123** [comment]
> Thanks for the bug report. I'll look into this soon.

How to play only one of the 4-voice, or play two at the same time, now four sounds can be played at the same time.

**Comment by watersoft123** [comment]
I use abcjs in Android webview control, it can play but can not see the button of play/pause and other. 
![Uploading Screenshot_20210331_071859_edit_1050087059614768.jpg…]()

**Comment by watersoft123** [comment]
![Uploading Screenshot_20210331_071859_edit_1050869710158398.jpg…]()

**Comment by paulrosen** [comment]
For playing only one voice, see the example https://cdn.rawgit.com/paulrosen/abcjs/master/examples/karaoke-synth.html

**Comment by paulrosen** [comment]
For the buttons - sorry I fixed that bug haven't updated my site yet. I'll do that this weekend.

**Comment by watersoft123** [comment]
> For the buttons - sorry I fixed that bug haven't updated my site yet. I'll do that this weekend.

Thanks Paul. The example of karaoke is very good.

**Comment by flreuss** [comment]
Feel like this issue got hijacked a bit... What happened to the issue I initially reported?

**Comment by paulrosen** [comment]
I'm currently in the middle of it! I'm tackling all the wrapping issues I know about. It shouldn't be long before I have another beta.

**Comment by paulrosen** [comment]
Looking at this closer - it looks like the notes are all present, but the stem directions are wrong.

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.33
